### DISCUSSION TITLE
Binding of child nodes.

### DISCUSSION THREAD

**Discussion by LukasSekerak** [eckig/graph-editor#147] [General]
Hi, i have TreeSkin structure and i want to implemet something like a "binding of nodes" - that should bind two nodes (only child nodes of one parent) together and this group of nodes should have only one connection to parent node - it is possible? Can you help me?

**Comment by eckig** [comment]
If I understand this correctly, you are talking about sub-graphs (graph in a graph)?

**Comment by LukasSekerak** [comment]
Actually no, i need to bind two nodes together - and want that they are be dragable together and have one connector to parent....
